concepts/05 UI Components/Button/00 Getting Started with Button/12 Add an Icon.md

```diff
@@ -0,0 +1,52 @@
+The [icon](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#icon) property adds a glyph to the button. You can load a custom icon or display one of the ready-to-use images from [multiple supported image sources](/Documentation/Guide/Themes_and_Styles/Icons/). The sample below shows how to display a glyph from the [DevExtreme library](/Documentation/Guide/Themes_and_Styles/Icons/#Built-In_Icon_Library):
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#button").dxButton({
+            // ...
+            icon: "comment"
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-button ...
+        icon="comment">
+    </dx-button>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxButton ...
+            icon="comment"
+        />
+    </template>
+
+##### React
+
+    <!-- tab: App.js -->
+    // ...
+
+    function App() {
+        return (
+            <div className="App">
+                <Button ...
+                    icon="comment"
+                />
+            </div>
+        );
+    }
+
+    export default App;
+
+---
+
+You have configured basic Button features. To take a more detailed look at this UI component, explore the following resources:
```

Comment:
- "You have configured basic Button features" Why do you add this? This is rather obvious since this was the point of the tutorial. 
- "To take a more detailed look at this UI component, explore the following resources" A more detailed look? What does this imply? There is more relevant information about this component? If so, why is this information not included in this topic?